docs/content/windows/components/Checkbox.mdx

```diff
@@ -24,6 +24,7 @@ Checkbox supports the following tokens:
   2. checkboxBorderColor – This changes the border color of the Checkbox.
   3. checkmarkColor – This changes the color of Checkmark.
   4. borderRadius - This changes the border radius of the Checkbox (use this to create a circular checkbox)
+  5. checkmarkVisibility - Allows you to change Checkmark visibility - 1 is visible. More info in Examples below.
```

Comment:
j.w. are there other expected scenarios besides the hover checkbox?  does the hover checkbox show the check in different colors for hover vs. committed vs committed + hover?

Also these docs changes should go into the microsoft/fluentui-site repo; I would link the PR there from this PR description

Reply:
This token allows you to change the Opacity of the Checkmark, there's another token that allows you to change the color of the token

Reply:
Linked the Docs in this PRs description